styles.css

```diff
@@ -21,5 +21,48 @@
     width: 100%;
     margin-top: 30px;
     display: flex;
-    justify-content: space-evenly
-}
\ No newline at end of file
+    justify-content: space-evenly;
+}
+
+.activityFirst {
+    margin-top: 150px;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+}
+
+.container {
+    width: 45%;
+    display: flex;
+    align-items: center;
+    border: 2px solid black;
+    border-radius: 5px;
+}
+
+.numeration {
+    margin-top: -230px;
+}
```

Comment:
You can try this `margin-left: 5px;` to slightly separate the numbers from the left edge